test/e2e/codecept.conf.js

```diff
@@ -5,22 +5,39 @@ exports.config = {
   output: './output',
   timeout: 1000,
   helpers: {
-    Nightmare: {
-      url: config.get('e2e.frontendUrl'),
+    Puppeteer: {
+      url: process.env.FRONTEND_URL || 'https://localhost:3000',
```

Comment:
This can be left as `config.get('e2e.frontendUrl')` as it does the same thing as above.